File: dev-docs/snippets/document-blob/pause-upload.ts
import type { JsBaoClient } from "js-bao-wss-client";

export function pauseUpload(
  client: JsBaoClient,
  documentId: string,
  blobId: string,
) {
  // #region example
  const blobs = client.document(documentId).blobs();
  const paused = blobs.pauseUpload(blobId); // true if an in-progress upload was paused
  // #endregion example
  return paused;
}

File: dev-docs/snippets/document-blob/pause-upload.swift
import JsBaoClient

func pauseUpload(
  client: JsBaoClient,
  documentId: String,
  blobId: String
) {
  // #region example
  let blobs = client.documents.blobs(documentId: documentId)
  let paused = blobs.pauseUpload(blobId: blobId)  // true if an in-progress upload was paused
  // #endregion example
  _ = paused
}
